src/backend/webGlObjects/baseWebGlObject.ts

```diff
@@ -5,6 +5,8 @@ export type WebGlObjectTag = {
     customData?: any;
 };
 
+export type WebGlObjectSource = string[];
```

Comment:
Instead, this should be an object which also stores *when* the object was created.
There could also be a flag where even removed objects are listed / kept, but their `delete` location / time is also recorded.